lib/vquic/vquic.c

```
@@ -428,9 +428,9 @@ static CURLcode recvmmsg_packets(struct Curl_cfilter *cf,
       }
       if(!cf->connected && SOCKERRNO == SOCKECONNREFUSED) {
         struct ip_quadruple ip;
-        Curl_cf_socket_peek(cf->next, data, NULL, NULL, &ip);
-        failf(data, "QUIC: connection to %s port %u refused",
-              ip.remote_ip, ip.remote_port);
+        if(!Curl_cf_socket_peek(cf->next, data, NULL, NULL, &ip))
```

We suppress the failure message here. Is that what we want?